var/exp_repo/experiments/laghos/experiment.py

```diff
@@ -0,0 +1,62 @@
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Laghos(Experiment):
+
+    variant(
+        "experiment",
+        default="example",
+        values=("weak", "strong","example"),
+        description="weak or strong scaling",
+    )
+
+    def compute_applications_section(self):
+        variables = {}
+            
+        if self.spec.satisfies("scaling=weak"):
+            variables["rs"] = ["5","6"]
+            variables["n_nodes"] = ["1","8"]
+        elif self.spec.satisfies("scaling=strong"):
+            variables["n_nodes"] = ["1","8"]
+            variables["rs"] = "5"
+        else:
+            variables["n_nodes"] = ["1","2","4","8","16","32","64","128"]    
+        variables["n_ranks"] = "{sys_cores_per_node} * {n_nodes}"
```

Comment:
Since we're setting ranks explicitly above let's move both n_nodes and n_ranks into the else, and set this to one node, 8 ranks since this will be for the example run.